app/errors.py

```
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine-related errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Exception raised when a visitor is not vaccinated."""
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    """Exception raised when a visitor's vaccine is expired."""
+    pass
+
+
+class NotWearingMaskError(Exception):

```

Consider making `NotWearingMaskError` inherit from `VaccineError` to follow the task description, which states that `VaccineError` should be a parent class for `NotVaccinatedError` and `OutdatedVaccineError`. While wearing a mask is not directly related to vaccines, grouping it under `VaccineError` would allow the `go_to_cafe` function in `main.py` to catch all exceptions related to cafe entry requirements in one `except` block, as suggested in the task description.

```python
class NotWearingMaskError(VaccineError):
    """Exception raised when a visitor is not wearing a mask."""
    pass
```